generators/app/templates/app/controllers/admin/JWTBlacklist.ts

```diff
@@ -0,0 +1,11 @@
+import { Controller } from "@/libraries/routes/decorators";
```

Comment:
I don't think we need to expose this to the Admin by default, JWTBlacklist is part of the internal auth logic

Reply:
It's true, I will remove it